src/index.js

```diff
@@ -555,6 +556,7 @@ function createConfig(options, entry, format, writeMeta) {
 								},
 							},
 							tsconfig: options.tsconfig,
+							objectHashIgnoreUnknownHack: !!options.closure,
```

Comment:
what's that?

Reply:
Without it async functions won't work with the ts plugin when combined with closure

https://github.com/ezolenko/rollup-plugin-typescript2/issues/105

Reply:
Just randomly saw this linked, but per the last comment there (from me), it's no longer necessary if you update to v0.26.0+ .

And per the other comments / former docs, if you are using the `objectHashIgnoreUnknownHack` option, you should really use it with `clean: true` as otherwise it can cause caching issues.